src/backend/distributed/commands/index_pg_source.c

```diff
@@ -181,13 +181,13 @@ ChooseIndexColumnNames(List *indexElems)
 			{
 				break;          /* found nonconflicting name */
 			}
-			sprintf(nbuf, "%d", i);
+			sprintf(nbuf, "%d", i); /* lgtm[cpp/banned-api-usage-required-any] */
```

Comment:
This is a function copied from PG codebase. I think it is better to keep them as is, and suppress messages.